src/SDKs/EventGrid/DataPlane/Microsoft.Azure.EventGrid/Microsoft.Azure.EventGrid.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<!-- Please do not move/edit code below this line -->
+	<Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+	<!-- Please do not move/edit code below this line -->
+
+	<PropertyGroup>
+		<PackageId>Microsoft.Azure.EventGrid</PackageId>
+		<Description>Provides developers with a library to publish events to Azure Event Grid. </Description>
+		<Version>1.0.0-preview</Version>
+		<AssemblyName>Microsoft.Azure.EventGrid</AssemblyName>
+		<PackageTags>Microsoft Azure EventGrid;Event Grid;Event Grid Publishing</PackageTags>
+		<PackageReleaseNotes>First version of Azure EventGrid library</PackageReleaseNotes>
+	</PropertyGroup>
+	<PropertyGroup>
+		<TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+	</PropertyGroup>
+
+	<!-- Please do not move/edit code below this line -->
+	<Import Condition="Exists('[MSBuild]::GetPathOfFileAbove(AzSdk.RP.props)')" Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
+	<ItemGroup>
```

Comment:
@kalyanaj please remove line 20 - 24 from the project file.
Also please end all package tags with a semicolon.